.dev/scripts/copyRuntime.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+source .env
+CONNECTORPATH=$(pwd)
+
+if [ -z "$RUNTIME_PATH" ]; then
+    echo "Please provide the path via the .env in the root folder file"
+    exit 1
+fi
+
+currentPackageName=$(jq .name package.json 2>/dev/null | tr -d '"') || ""
+if [ -z "$currentPackageName" ] || [ "$currentPackageName" != "@nmshd/connector" ]; then
+    echo "Please run this script from the root of the connector"
+    exit 1
+fi
+
+## declare an array variable
+declare -a arr=("consumption" "content" "iql" "runtime" "transport")
+
+## now loop through the above array
+for package in "${arr[@]}"; do
+    cd "$RUNTIME_PATH/packages/$package" || exit
+    packageName=$(jq .name package.json | tr -d '"')
+    # printf "Copying $packageName to $currentPackageName \n"
+    cp dist $CONNECTORPATH/node_modules/$packageName -r
```

Comment:
Dumb question: can't you just symlink the directories? Would make the nodemon script obsolete, too.

Reply:
Sadly no, in theories it is working, but the linked packages would use their own node_modules folder. And then we run into problems with the typescript-IOC (dependency injection) package because there are two instances of it, the dependency injection from the runtime to the connector is not working anymore.

Reply:
Hm .. weird that it would work with a copy then. Would you mind showing me this on Monday? Maybe we'll figure something out.